profiler/shakesapp/README.md

```diff
@@ -0,0 +1,32 @@
+# Profiler Shakespeare Application in Go
+
+This application starts a server which will return the number of times a word or
```

Comment:
Do we want to mention that this app is intentionally non-optimal and the goal of it is demonstrate how to make a set of progressive steps towards optimizing it? I would also include the exact progressive versions in subdirectories, like we do in the internal version. (just in case - let's not use Git branches for that).

Reply:
Stated that this application is non-optimal.